packages/paste-website/src/pages/components/modal/index.mdx

```diff
@@ -2,6 +2,7 @@
 title: Modal
 package: '@twilio-paste/modal'
 description: A modal displays content over the main page and blocks any interaction with the page.
+slug: /components/modal/
```

Comment:
Is there no other way to generate this? From the graphql maybe? This could be error prone..

Reply:
Ok